allauth/templates/account/email_change.html

```diff
@@ -16,14 +16,14 @@
         {% slot body %}
             {% csrf_token %}
             {% if current_emailaddress %}
-                {% element field disabled=True type="email" value=current_emailaddress.email %}
+                {% element field id="current_emailaddress" disabled=True type="email" value=current_emailaddress.email %}
                     {% slot label %}
                         {% translate "Current email" %}:
                     {% endslot %}
                 {% endelement %}
             {% endif %}
             {% if new_emailaddress %}
-                {% element field name="email" value=new_emailaddress.email disabled=True type="email" %}
+                {% element field id="new_emailaddress" value=new_emailaddress.email disabled=True type="email" %}
```

Comment:
Note that I removed the `name="email"` from this disabled element because that results in two inputs named "email" on the same form.  A disabled input shouldn't get sent with the form submission either way, but could be confusing.